tools/cldr/cldr-to-icu/src/main/java/org/unicode/icu/tool/cldrtoicu/ant/GenerateCodeTask.java

```diff
@@ -0,0 +1,93 @@
+// © 2019 and later: Unicode, Inc. and others.
+// License & terms of use: http://www.unicode.org/copyright.html
+package org.unicode.icu.tool.cldrtoicu.ant;
+
+import main.java.org.unicode.icu.tool.cldrtoicu.CodeGenerator;
+import main.java.org.unicode.icu.tool.cldrtoicu.generator.ResourceFallbackCodeGenerator;
+import org.apache.tools.ant.BuildException;
+import org.apache.tools.ant.Task;
+
+import java.io.*;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+// Note: Auto-magical Ant methods are listed as "unused" by IDEs, unless the warning is suppressed.
+public final class GenerateCodeTask extends Task {
+    private Path cldrPath;
+    private File cOutDir;
+    private File javaOutDir;
```

Comment:
Curious: Why do you use both the newer Path and the ancient File?
https://www.baeldung.com/java-path-vs-file

Reply:
Ignorance.  I didn't know how the new Path object worked and couldn't find methods on it that did what I wanted.  Thanks for the pointer to the article; that helped a lot.  I've updated this class to use Path everywhere.